### DISCUSSION TITLE
Firefox GX logo

### DISCUSSION THREAD

**Discussion by Maxterino** [Godiesc/firefox-gx#49] [General]
I would like to replace my stock firefox logo with the red and blue Firefox GX logo so it matches the theme. I can't find the logo anywhere. Can somebody link/send me the logo? Preferred in SVG but PNG is fine too

**Comment by Godiesc** [comment]
There are 2 logos in `chrome>images`, foxlogo.svg and foxlogo2.svg, the first has the left color adapt follow the accent color of the theme you use, and the second has static colors, so you will want to use this.

**Reply by Maxterino** [reply]
found it. Thank you so much!
